vip-helpers/vip-utils.php

```diff
@@ -1230,6 +1230,29 @@ function wpcom_vip_add_URI_to_newrelic(){
 add_action( 'muplugins_loaded', 'wpcom_vip_add_URI_to_newrelic' );
 
 /**
+ * Name cron correctly in New Relic and do not count it as part of the Apdex score
```

Comment:
We should mention "why" we don't want it to count in the score in the inline comment.